### PR TITLE
handle old format

### PR DESCRIPTION
**Pull request by snopoke**

### CODE REVIEW

**Comment by snopoke** [comment]
@TylerSheffels

File: corehq/apps/cleanup/management/commands/rebuild_cases_with_mismatched_forms.py [hunk 1]
```diff
@@ -14,7 +14,10 @@ def wrapper(row):
         id = row["id"]
         doc = ExceptionRecord.get(id)
         domain = doc.domain if hasattr(doc, "domain") else ""
-        domain, case_id = domain.split(',')
+        try:
+            domain, case_id = domain.split(',')
+        except ValueError:
+            return {'exception': doc}
```
Comment (sravfeyn) [inline corehq/apps/cleanup/management/commands/rebuild_cases_with_mismatched_forms.py]: `exception` key name is little misleading.

Comment (snopoke) [inline corehq/apps/cleanup/management/commands/rebuild_cases_with_mismatched_forms.py]: suggestion?

Comment (sravfeyn) [inline corehq/apps/cleanup/management/commands/rebuild_cases_with_mismatched_forms.py]: ha! It is an ExceptionRecord doc, I thought it is some old domain doc. Could change it to `exception_doc`. But this is fine too.

**Pull request merged**